nexus/db-model/src/vpc.rs

```diff
@@ -110,6 +110,14 @@ impl IncompleteVpc {
             subnet_gen: Generation::new(),
         })
     }
+
+    /// Create a copy of self, but with a new random VNI.
+    ///
+    /// This is used to retry insertion of a VPC in the case where we can't find
+    /// an available VNI.
+    pub fn with_new_vni(self) -> Self {
```

Comment:
Super nitpick, this is `with_random_vni`, as it could randomly be the same VNI as before

Reply:
Done in ca6d7ccdf